airbyte-scheduler/src/main/java/io/airbyte/scheduler/SchedulerApp.java

```diff
@@ -146,6 +149,20 @@ public static void main(String[] args) {
         configs.getAirbyteVersion(),
         configRepository);
 
+    Optional<String> airbyteDatabaseVersion = jobPersistence.getVersion();
+    int loopCount = 0;
+    while (airbyteDatabaseVersion.isEmpty() && loopCount < 300) {
```

Comment:
Scheduler App is waiting for server here... 
Are the thresholds for the loop ok?

Reply:
yes